app/src/main/java/org/oppia/app/activity/ActivityComponent.kt

```diff
@@ -58,7 +58,7 @@ interface ActivityComponent {
   fun inject(bindableAdapterTestActivity: BindableAdapterTestActivity)
   fun inject(conceptCardFragmentTestActivity: ConceptCardFragmentTestActivity)
   fun inject(contentCardTestActivity: ContentCardTestActivity)
-  fun inject(continuePlayingActivity: ContinuePlayingActivity)
+  fun inject(recentlyPlayedActivity: RecentlyPlayedActivity)
```

Comment:
We place these methods in alphabetical order. Please make this change.

Reply:
@abhinavraj23 Please reply to this comment and everywhere else. Because these are simple changes you can directly write "Done"